src/components/addCohort/addCohort.js

```diff
@@ -0,0 +1,17 @@
+import React from "react-dom";
+import { useState } from "react";
+// import Button from " @material-ui/core/Button";
+
+export default function AddCohort() {
+  const [cohort, setCohort] = useState("");
+
+  const handleChange = () => {
+    console.log("read event: ");
+  };
+  return (
+    <>
+      <button onclick={() => handleChange()}>Add Cohort</button>
+      {/* <Button variant="outlined">Default</Button> */}
```

Comment:
Is this needed anymore? If not, you can remove the commented code.

Reply:
File removed.